pkg/apis/devops/v1alpha2/register.go

```diff
@@ -271,27 +271,27 @@ func addWebService(c *restful.Container) error {
 		Param(webservice.QueryParameter("branch", "branch ").
 			Required(false).
 			DataFormat("branch=%s")).
-		Returns(http.StatusOK, RespOK, []devops.PipelineRun{}).
-		Writes([]devops.PipelineRun{}))
+		Returns(http.StatusOK, RespOK, []devops.PipeBranchRun{}).
+		Writes([]devops.PipeBranchRun{}))
```

Comment:
`PipeBranch` seems like a new word, also not descriptive, I suggest use `PipelineBranch`

Reply:
I think it is more appropriate to use `BranchPipeline`.